src/tools/rest-frontend/resources/grunt-tasks/grunt-create-website-news.js

```diff
@@ -57,7 +57,6 @@ module.exports = function (grunt) {
                     shortDesc = shortDesc[1];
                 } else {
                     shortDesc = '';
-                    grunt.log.warn('News post ' + post.name + ' has no description matching the configured regex');
```

Comment:
The warning seems to be useful? Could you please also document where logs are written to?